state_handlers.js

```diff
@@ -25,36 +28,49 @@ var state_handlers = {
         this.emit(':responseReady');
       },
       SearchAndPlayArticle: function() {
+        console.log('START_MODE:SearchAndPlayArticle');
+        synthesisHelper(this);
+      },
+      SearchAndSummarizeArticle: function() {
+        console.log('START_MODE:SearchAndSummarizeArticle');
         synthesisHelper(this);
       },
       ScoutMyPocketSummary: function() {
+        console.log('START_MODE:ScoutMyPocketSummary');
         synthesisHelper(this);
       },
       ScoutHeadlines: function() {
+        console.log('START_MODE:ScoutHeadlines');
         synthesisHelper(this);
       },
       ScoutTitles: function() {
+        console.log('START_MODE:ScoutTitles');
         getTitlesHelper(this);
       },
       'AMAZON.HelpIntent': function() {
+        console.log('START_MODE:AMAZON.HelpIntent');
+
         var message = 'Welcome to Scout. You can say, get titles to begin.';
         this.response.speak(message).listen(message);
         this.emit(':responseReady');
       },
       'AMAZON.StopIntent': function() {
-        var message = 'Good bye.';
-        this.response.speak(message);
+        console.log('START_MODE:AMAZON.StopIntent');
+        this.response.speak(ALEXA_STOP_RESP);
```

Comment:
Need to scope this one `constants.errString.ALEXA_STOP_RESP`

Reply:
good catch.